cocos/ui/UIPageViewIndicator.h

```diff
@@ -79,9 +83,11 @@ class PageViewIndicator : public ProtectedNode
     PageView::Direction _direction;
     Vector<Sprite*> _indexNodes;
     Sprite* _currentIndexNode;
+	Sprite* _currentOverlappingIndexNode;
```

Comment:
In the current implementation of the class, the "selected" index node simply places itself on top of the corresponding "unselected" index node.  However, both nodes are still visible, which can lead to ugly artifacts around the edges of the "selected" circle, since the "unselected" one is still visible behind it.  Also, if the user sets the "selected" node's opacity to e.g. 50%, the result will look wrong because there will still be the "unselected" node showing through behind it.

My code changes this, and sets the "unselected" node invisible when the selected one is on top of it; hence, we need a pointer to it, so that when the selection changes, the previous overlapped node can be set visible again.